### PR TITLE
Fix reload when switching networks on the browser

### LINKED ISSUE #3078: On switching account, network - the browser (and site) doesn't update right away
### Situation
Using a site in the in-app browser

### Action (s) that lead to the issue
Respond to a connect request or switch account or switch network

### Expected behavior
The action made on MetaMask should be reflected immediately in the site

### Buggy behavior
No response is reflected on the site / browser. You have to manually refresh (if you figure it out) for the change to be reflected.

### PR DESCRIPTION
**Pull request by andrepimenta**
**Description**

This PR fixes the reload when switching networks on the browser.

**Issue**

Resolves #3078

### CODE REVIEW

**Review by gantunesr — COMMENTED** [review]
I left a question regarding the Podfile

File: ios/Podfile.lock [hunk 1]
```diff
@@ -661,7 +661,7 @@ SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   Branch: 6a281514287f99d707615ac62c2cca69e0213df0
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
-  DoubleConversion: cde416483dac037923206447da6e1454df403714
+  DoubleConversion: 5805e889d232975c086db112ece9ed034df7a0b2
```
[the remark this answers is not in the document]
Comment (andrepimenta) [reply ios/Podfile.lock]: Was resolved with merging develop back

**Review by cortisiko — APPROVED** [review]
🌮 🌮  🌮  🌮

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**